lib/pbench/test/unit/server/conftest.py

```diff
@@ -30,6 +30,11 @@
 from pbench.server.database.models.users import User
 from pbench.test import on_disk_config
 from pbench.test.unit.server.headertypes import HeaderTypes
+from pbench.test.unit.server.query_apis.test_query_builder import (
+    ADMIN_ID,
+    SELF_ID,
+    USER_ID,
+)
```

Comment:
Now that virtually all of our unit test code depends on these blind assumptions, I think the constants should be *moved* here (or into a module `__init__.py`) and *imported* by `test_query_builder.py`.

This plays into the forthcoming OIDC migration, where the ID values will definitely be changing and the representation of the users will be vastly different as well. (Which could be taken as a suggestion that meddling in this area might be "unwise".)

Oh, and, in the larger context, `SELF` and `USER` are pretty much meaningless, so we should at least have better names. 😜

Reply:
> I think the constants should be moved here

Bummer.  I was hoping to get away with more of a point fix.  (I would have liked to have requested this fix in #3221, but it merged before I could....)

> Oh, and, in the larger context, SELF and USER are pretty much meaningless, so we should at least have better names. 😜

I guess I'm open to suggestions, then....

Reply:
`DRB_ID`, `TEST_ID`?? Not fancy, but at least vaguely meaningful in context. Or you could go all out and try `DRB_USER_ID`, `TEST_USER_ID`, and `ADMIN_USER_ID` if you find some extra letters raining from the sky ...